nextcloud/mail
# feat: classify emails by importance based on subjects

Partly addresses https://github.com/nextcloud/mail/issues/3968

Closes https://github.com/nextcloud/mail/pull/8257

Part 5: The great rebasing ...

## Improvements

- Classify emails by importance using a new and improved classification pipeline
- Persist classifier models (and transformer pipelines) in memory cache only
- Drop the `oc_mail_classifiers` table

## How to tests?

1. Have some important mails in your inbox.
2. Make sure to have the classification enabled: Account settings modal -> "Data collection consent" :white_check_mark: and "Mark as important" :white_check_mark:  
3. Train a model: `occ mail:account:train -vvv <account-id>` (extract account id with `occ mail:account:export <user-id>`)
4. Receive some mails and observe that some are marked as important and some aren't.
